tests/upgrade/upgrade_test.go

```diff
@@ -0,0 +1,63 @@
+package tests
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	. "github.com/portworx/torpedo/tests"
+)
+
+func TestUpgrade(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Torpedo : Upgrade")
+}
+
+var _ = BeforeSuite(func() {
+	InitInstance()
+})
+
+var _ = Describe("Upgrade volume driver", func() {
+	It("upgrade volume driver and ensure everything is running fine", func() {
+		contexts := ScheduleAndValidate("upgradevolumedriver")
+
+		Step("start the upgrade of volume driver", func() {
+			err := Inst().V.UpgradeDriver("1.2.11.4")
```

Comment:
Can the version be configurable? May be use env variables OR pass in to the upgrade_test binary as args ?